docs/blog/posts/2023-08-23-howRenvJupyterKernel.md

```diff
@@ -1,25 +1,34 @@
 ---
 date:
   created: 2023-08-23
+categories:
+    - R
+    - Environments
+authors:
+    - mason
 ---
 
 # How do I export my R environment to JupyterHub?
 
 If you've created an `R` virtual environment and want to use it as a kernel in JupyterHub, follow the steps below.
 
-{% include note.html content="Want to use a Python virtual environment in JupyterHub? Have a look at <a href=\"/training/6_jupyter_hub.html\">this page</a>"%}
+
+!!! note
+    Want to use a Python virtual environment in JupyterHub? Have a look at [this page](/_getting_started/jupyter/){target="_blank"}
+
+<!-- more -->
 
 ### Launch `R` in Virtual Environment
 To start, open a terminal window, SSH into the Yens, and launch the instance of `R` associated with your virtual environment.
 
-```bash
-$ /path/to/virtual/environment/bin/R
+```title="Terminal Command"
+/path/to/virtual/environment/bin/R
 ```
 
 ### Install and Use `IRkernel`
 Once in your instance of `R`, install the `IRkernel` package and create a Jupyter kernel for your `R` environment.
 
-```R
+```{ .yaml .no-copy title="Terminal Output"}
```

Comment:
you can add .r to syntax highlight the output with R language

Reply:
done